src/main/java/org/zalando/nakadi/plugin/auth/DefaultAuthorizationService.java

```diff
@@ -19,6 +24,14 @@ public boolean isAuthorizationAttributeValid(final AuthorizationAttribute author
         return true;
     }
 
+    @Override
+    @Nullable
+    public Optional<Subject> getSubject() {
+        return Optional.of(
+                () -> Optional.ofNullable(SecurityContextHolder.getContext().getAuthentication())
+                        .map(Principal::getName).orElse("unauthorized"));
```

Comment:
Shouldn't this just return Optional.empty()? which is then handled in other places?

Reply:
Right, fixed in `3350ff9`. That makes sense. Thanks :)